scripts/release-pypath/builder/homebrew.py

```diff
@@ -72,8 +72,9 @@ def from_dependencies(
             if pkg.name == "dbt":
                 dbt_package = pkg
             # we can assume that anything starting with dbt- in a fresh
-            # venv is a dbt package, I hope
-            elif pkg.name.startswith("dbt-"):
+            # venv is a dbt package, except dbt-extractor.
+            # when adapter plugins get split up, this janky check will go away.
+            elif pkg.name.startswith("dbt-") and pkg.name is not "dbt-extractor":
```

Comment:
```suggestion
            elif pkg.name.startswith("dbt-") and pkg.name != "dbt-extractor":
```
🤦 this is my bad, we are comparing strings here

Reply:
also my bad for copy pasta-ing this 😂